chapters/annotations.tex

```diff
@@ -2052,24 +2055,26 @@ \subsection{Protection of Classes}\label{protection-of-classes}
   Same as \lstinline!Access.packageText! and additionally the class, or part of the class, can be copied.
 \end{enumerate}
 
-The \lstinline!access! annotation holds for the respective class and all classes
-that are hierarchically on a lower level, unless overridden by a
-\lstinline!Protection! annotation with \lstinline!access!.
-Overriding \lstinline!access=Access.hide! or \lstinline!access=Access.packageDuplicate!
-has no effect.
+The \lstinline!access! annotation holds for the respective class and all classes that are hierarchically on a lower level, unless overridden by a \lstinline!Protection! annotation with \lstinline!access!.
+Overriding \lstinline!access=Access.hide! or \lstinline!access=Access.packageDuplicate! has no effect.
+
+Classes should not use other classes in ways that contradict this protection.
+Tools must ensure that protected contents is not shown, even if classes do not meet this requirement.
```

Comment:
```suggestion
Tools must ensure that protected content is not shown, even if classes do not meet this requirement.
```

Reply:
I'm not sure if that change is necessary, but I'm not totally against it.

"Contents" implies that it is more tangible and countable, and to me the individual elements are so tangible and countable and none of them should be shown.

However, I can understand "content" in the sense that it is the protected ideas represented there that shouldn't be shown.

Reply:
Yes. Done.